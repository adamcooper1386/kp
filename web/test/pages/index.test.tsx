import React from 'react'
import { render } from '../testUtils'
import AllPostsDataMock from '../__mocks__/allPostsDataMock'
import Home from '../../src/pages/index'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Home />)
    //allPostsData={AllPostsDataMock.emptyAllPostsData}/>, {})
    expect(asFragment()).toMatchSnapshot()
  })

  // it('clicking button triggers alert', () => {
  //   const { getByText } = render(<Home />, {})
  //   window.alert = jest.fn()
  //   fireEvent.click(getByText('Test Button'))
  //   expect(window.alert).toHaveBeenCalledWith('With typescript and Jest')
  // })
})
