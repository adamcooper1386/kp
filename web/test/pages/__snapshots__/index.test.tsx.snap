// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <main>
      <section
        class="headingMd"
      >
        <h1>
          Ketosistant
        </h1>
        <h2>
          A modern application for managing medical keto diets
        </h2>
      </section>
      <section
        class="headingMd padding1px"
      >
        <h2
          class="headingLg"
        >
          We are under construction
        </h2>
        <p>
          We are working to bring this application to public availability as quickly as we can. We appreciate your patience in the interim.
        </p>
      </section>
    </main>
  </div>
</DocumentFragment>
`;
